docker-compose.yml

```diff
@@ -161,7 +164,7 @@ services:
     build: ./merchant
     ports:
       - "5003:5003"
-    command: python3 merchant.py --port 5003
+    command: python3 -u merchant.py --port 5003
```

Comment:
What does -u do?

Reply:
-u uses unbuffered output
Without that parameter, it can take a minute or more to see the merchant output in the docker-compose output